hibernate-reactive-core/src/main/java/org/hibernate/reactive/mutiny/Mutiny.java

```diff
@@ -1464,6 +1486,30 @@ default <T> Uni<T> get(Class<T> entityClass, Object id, LockModeType lockModeTyp
 		 */
 		<R> SelectionQuery<R> createQuery(String queryString, Class<R> resultType);
```

Comment:
Shouldn't this also have a `@Deprecated` (with link to createSelectionQuery)?

Reply:
No that one is OK. It's the ones without a `resultType` that are the problems.

Reply:
(I mean, it's _redundant_, but no need to deprecate it, IMO.)